src/configure/resources/messages.ts

```diff
@@ -52,4 +52,11 @@ export class Messages {
     public static azureServicePrincipalFailedMessage: string =`Failed while creating Azure service principal.`;
     public static roleAssignmentFailedMessage: string =`Failed while role assignement.`;
     public static waitForAzureSignIn: string =`Waiting for Azure sign-in...`;
+    public static userCancelledExcecption = 'User cancelled the action';
+    public static deploymentCenterAlreadyConfigured = 'Deployment source of type: %s is already setup for your web app, you can view more by browsing to deployment center.';
+    public static cannotFindPipelineUrlInMetaDataException = 'We were unable to find pipeline url for the app service. This can be caused due to corrupt/invalid metadata of site. You can open deployment center for more information.';
+    public static cannotFindOrganizationWithName = 'Unable to find organization with name: %s';
+    public static browseNotAvailableConfigurePipeline = 'Pipeline is not setup on the Azure resource. You can configure pipeline by clicking the below button.';
```

Comment:
Unable to find a pipeline for this Azure Web App. Please click on "Configure Pipeline" to setup.